src/helpers/getOrgIdAndEntityTypewithEntitiesBasedOnPolicy.js

```diff
@@ -201,7 +200,9 @@ module.exports = class OrganizationAndEntityTypePolicyHelper {
 					[Op.in]: entityTypes,
 				}
 			}
-
+			if (report_filter) {
```

Comment:
@sumanvpacewisdom pass the filter from the calling function and append directly to the filter, remove this condition